cmake/KPXCHelpers.cmake

```diff
@@ -1,33 +1,61 @@
-function(kpxc_run_macdeployqt_on_installed_helper_binary app_name installed_binary_relative_path)
-    # Running macdeployqt on a POST_BUILD copied binary is not useful for CPack, because
-    # the copied binary will be overriden by the corresponding install(TARGETS) command of the same
-    # binary.
-    # Thus, we run macdeployqt using install(CODE) on the already installed binary.
-    # The precondition is that install(TARGETS) has to be called before this function is called.
+# Running macdeployqt on a POST_BUILD copied binaries is pointless when using CPack because
+# the copied binaries will be overriden by the corresponding install(TARGETS) commands.
+# That's why we run macdeployqt using install(CODE) on the already installed binaries.
+# The precondition is that all install(TARGETS) calls have to be called before this function is
+# called.
+# macdeloyqt is called only once, but it is given all executables that should be processed.
+function(kpxc_run_macdeployqt_at_install_time)
+    set(no_value_options)
+    set(single_value_options
+        APP_NAME
+    )
+    set(multi_value_options
+        EXTRA_BINARIES
+    )
```

Comment:
We are using UPPERCASE (or sometimes PascalCase) for variable names everywhere else. This should be no exception.

Reply:
Are you referring to the names of no_value_options / single_value_options / multi_value_options?

At least from what i've seen, the emerging style is to name cache vars SHOUTY_CASE and local variables snake_case or camelCase/PascalCase. 

Naming local variables SHOUTY_CASE comes from before CMake modern times, so not really sure why you prefer that. :)

Having said that, which variables do you prefer to be renamed? All the ones used in kpxc_run_macdeployqt_at_install_time?

Reply:
Yes and yes. I do prefer lowercase, but consistency >> preference.

Reply:
Done